src/nativeresources/processrc.awk

```diff
@@ -54,12 +56,14 @@ BEGIN {
         gsub(/" +"/, "", $0);
         # remove all terminating newlines from the string - the msgfmt fails on those
         # since it expects them to be at the end of the msgid as well
-        while(gsub(/\\n"/, "\"", $0)) {}
+        if (removeTerm == 1) {
```

Comment:
Why do we need the terminating newlines removal to be parametric? What is the problem with the ildasm resources here?